content/en/language_clients/rust/overview.md

```diff
@@ -0,0 +1,33 @@
+---
+type: docs
+category: Rust
+title: Rust Client Overview
+weight: 5
+---
+
+[`sigstore`](https://crates.io/crates/sigstore) is a crate designed to interact with Sigstore architecture.
+
+This crate is under active development, but will not be considered stable until the 1.0 release.
```

Comment:
Can we include a link to https://github.com/sigstore/sigstore-rs/issues/274 to track the 1.0 release?

Reply:
Good call. Updating now.